### ISSUE TITLE
Gradle Kotlin support

### ISSUE THREAD

**Issue by asm0dey** [tommyettinger/gdx-liftoff#127]
Could you please add the support for Gradle Kotlin DSL? It's at the same time better supported in IDEA and is now an [official default language for the new Gradle projects](https://blog.gradle.org/kotlin-dsl-is-now-the-default-for-new-gradle-builds).

I find it much more maintainable because the code autocompletion works fine, contrary to the Groovy DSL. Readability has improved a lot in the last few years.

**Comment by tommyettinger** [comment]
I know there are people who use it, though I really don't know very many who do. Supporting this would involve duplicating or replacing all Gradle file generation across the project. And not only that, it's already hard enough to provide instructions for new users when they could be using gdx-setup or gdx-liftoff. Imagine if one person needs gdx-setup (Gradle) help, another needs gdx-liftoff (Gradle) help, and a third needs gdx-liftoff (Kotlin) help. The instructions range from subtly different to totally incompatible.

> The Tower of Babel ([Hebrew](https://en.wikipedia.org/wiki/Hebrew_language): מִגְדַּל בָּבֶל‎, Mīgdal Bāḇel) narrative in [Genesis](https://en.wikipedia.org/wiki/Book_of_Genesis) 11:1–9 is an [origin myth](https://en.wikipedia.org/wiki/Origin_myth) and [parable](https://en.wikipedia.org/wiki/Parable) meant to explain why the world's peoples speak different languages.
> 
> According to the story, a united human race speaking a single language and migrating eastward, comes to the land of [Shinar](https://en.wikipedia.org/wiki/Shinar) (שִׁנְעָר‎). There they agree to build a city and a tower with its top in the sky. [Yahweh](https://en.wikipedia.org/wiki/Yahweh), observing their city and tower, confounds their speech so that they can no longer understand each other, and scatters them around the world.

In short, you aren't building anything big if you can't agree on a language... Right now only Kotlin megafans within the libGDX community seem to be the ones adopting the Kotlin DSL. Unless there suddenly appear many more Kotlin experts willing to help with mysterious and frustrating issues faced by new users, adding generation support for the Kotlin DSL is a wasted effort. A **large** wasted effort.

I'm closing this because I do not plan to implement this myself. If someone decided to PR this in a pristine, perfectly-wrapped package with a big red ribbon, I might merge it if I felt confident the PR submitter would still be around to fix things later. I am not suggesting anyone should go out and do that unless they really want to.

**Comment by asm0dey** [comment]
Fair enough, thanks
